install

```diff
@@ -96,3 +96,4 @@ echo "alias rosdevel='source ~/catkin_ws/devel/setup.bash' # needed for autocomp
 echo "alias rosvenv='source ~/catkin_ws/src/buzzmobile/bin/activate' # activate venv" >> ~/.bashrc
 echo "rosinit     # put these in your .bashrc too so they run everytime" >> ~/.bashrc
 echo "rosdevel    # you open a new terminal window/tab." >> ~/.bashrc
+echo "export SVGA_VGPU10=0" >> ~/.bashrc # disables support for OpenGL 3.3, allows gzclient to run
```

Comment:
Uhhh, shouldn't this be done as part of rosinit, and not just done all the time? or will this not cause unwanted side effects?